context.go

```diff
@@ -352,7 +352,9 @@ func (c *context) FormParams() (url.Values, error) {
 
 func (c *context) FormFile(name string) (*multipart.FileHeader, error) {
 	f, fh, err := c.request.FormFile(name)
-	defer f.Close()
+	if f != nil {
+		defer f.Close()
```

Comment:
Hi!
Is it better to just close the file without `defer`?

Reply:
You are right. Should I remove defer?

Reply:
I think you should. Because this version is a bit confusing.